plugin/src/main/resources/seedJobGroovyScript.groovy

```diff
@@ -74,7 +74,17 @@ def findParentItem(String fullName) {
         default:
             def parentName = fullName.substring(0, i)
             Item item = jenkins.getItemByFullName(parentName)
-            return item instanceof ItemGroup ? (ItemGroup) item : null
+
+            if (item == null) {
+                println "  WARNING: parent folder ${parentName} not found"
+                return null
+            }
```

Comment:
Please move the `else if` to this line for consistency, same with the `else` below.

Reply:
Done

Reply:
Done